client/packages/inventory/src/Stocktake/DetailView/SidePanel.tsx

```diff
@@ -52,30 +52,30 @@ const AdditionalInfoSection: FC = () => {
   );
 };
 
-export const SidePanel: FC = () => {
+export const SidePanel = () => {
   const { success } = useNotification();
   const t = useTranslation('inventory');
-  const { data } = useStocktake.document.get();
   const { mutateAsync } = useStocktake.document.delete();
-  const canDelete = data ? canDeleteStocktake(data) : false;
+  const { data: stocktake } = useStocktake.document.get();
+  const canDelete = stocktake ? canDeleteStocktake(stocktake) : false;
 
   const copyToClipboard = () => {
     navigator.clipboard
-      .writeText(JSON.stringify(data, null, 4) ?? '')
+      .writeText(JSON.stringify(stocktake, null, 4) ?? '')
       .then(() => success('Copied to clipboard successfully')());
```

Comment:
I'm getting something weird happening with copy to clipboard on android?


https://github.com/msupply-foundation/open-msupply/assets/8287373/ce27edb1-00bc-45b4-be41-ad52c4eb333d

Reply:
the function is not supported on android, and I thought the buttons were removed on the android platform.

Reply:
Want me to create an issue?